src/builder.rs

```diff
@@ -41,67 +41,252 @@ impl ExternalBuilder {
 		}
 	}
 
+	/// Sets Endpoint closure to prosess `pwasm_ethereum::call` to some Address
+	///
+	/// # Example
+	/// ```
+	/// extern crate pwasm_test;
```

Comment:
you might want to hide everything except the actual invocation of endpoint from doc

so only this:

```
+	/// 	ext_reset(|e| e
 +	///			.endpoint("0x16a0772b17ae004e6645e0e95bf50ad69498a34e".into(),
 +	///				Endpoint::new(Box::new(|_val, _input, result| {
 +	///					result[0] = 2;
 +	///					Ok(())
 +	///			})))
 +	/// 	);
 +	///		let mut result = [0u8; 1];
 +	///		let input = [2u8; 32];
 +	///		pwasm_ethereum::call(20000, &"0x16a0772b17ae004e6645e0e95bf50ad69498a34e".into(), 100.into(), &input, &mut result).unwrap();
 +	///		assert_eq!(result[0], 2);

``

makes sense to show

all other lines can be hidden with `#` in front